src/components/Calendar/Calendar.js

```diff
@@ -48,15 +57,30 @@ function CalendarInvite({ startDatetime, type = '', name, noMargin }) {
       : ''}
   `
 
+  // AddToCalendarHOC does not allow passing in additional arbitary links
+  // to render in the dropdown list. Instead of refactoring the external
+  // library, the Dropdown component was extend to support rendering
+  // additional elements from appendChildren & prependChildren props.
+  const expiryNotificationLink = (
+    <EmailNotifyLink key="email" domainName={name} address={registrant}>
+      {t('c.email')}
+    </EmailNotifyLink>
+  )
+
   const AddToCalendar = AddToCalendarHOC(CalendarButton, Dropdown)
   return (
     <AddToCalendar
       event={event}
       className={styles}
-      buttonText="Set renewal reminder"
+      buttonText={t('expiry.reminder')}
```

Comment:
Change this to a more specific namespace such as `expiryNotification` and add the other keys there too

Reply:
Updated